css-media-queries/meta-viewport.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <!-- your code here 👇 -->
+
+  <title>CSS Media Queries - Meta Viewport</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+
+    .container {
+      min-height: 100vh;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+
+    .box {
+      width: 200px;
+      height: 200px;
+      background-color: blue;
+    }
+
+    @media screen and (min-device-width: 568px) {
+      .box {
+        background-color: red;
+      }
+    }
+  </style>
+</head>
```

Comment:
Please try using the `<meta name="viewport" ... >` tag:

---

![image](https://user-images.githubusercontent.com/16143771/170392889-d901f506-c0bd-47db-a89e-de3580e313ce.png)